scripts/docker/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+#
+# Build file for the Dockerfile
+
+set -e 
+
+# DOCKER_REPOSITORY is the master repository where the docker image will be referenced and stored.  Defaults to "datafusionrs"
+DOCKER_REPOSITORY="${DOCKER_REPOSITORY:-datafusionrs}"
+
+# DOCKER_VERSION is the current version of the docker image.  Defaults to "0.1.4"
+DOCKER_VERSION="${DOCKER_VERSION:-0.1.4}"
```

Comment:
How do we make this pick up the version from cargo.toml? It would also be good to tag as `:lastest`

Reply:
If we pick it up from cargo.toml, we'd have to parse it.  If there's a config in there, I can add that.  Would you prefer this method?  Fairly easy to fix.